src/Doctrine/EntityClassGenerator.php

```diff
@@ -56,13 +56,14 @@ public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $
         $this->generateRepositoryClass(
             $repoClassDetails->getFullName(),
             $entityClassDetails->getFullName(),
-            $withPasswordUpgrade)
+            $withPasswordUpgrade,
+            $avoidDefaultComments)
```

Comment:
The `)` should be on the next line, right? It looks like that was wrong before this PR :). Can you fix it?

Reply:
Done